### ISSUE TITLE
404 error on "Add Blog item"

### ISSUE THREAD

**Issue by zaszlo** [getgrav/grav-plugin-admin#2182]
url: http://localhost/myblog/admin/pages/blog/title/:add

Will get me this displayed:
This page will not exist until it is saved.
Error 404
Woops! Looks like this page doesn't exist.

No errors are present in the grav.log.

Grav v1.7.13 - Admin v1.10.13

Thanks for looking into this, please ask if I should provide some more info.

**Comment by mahagr** [comment]
Hmm.. Maybe the blog page doesn't exist? You cannot add pages if the parent of the page doesn't exist, so you need to create that first.

**Comment by zaszlo** [comment]
You are probably right, because the my one and only blog on the site is not called blog but news... so I suppose I will have to correct add this information for the admin plugin, but how? Thanks for pointing this out.

**Comment by mahagr** [comment]
I think there may be a blueprint for this in either in a plugin or theme.

**Comment by zaszlo** [comment]
Yes, editing gravsite\user\blueprints\admin\pages\new_blog_item.yaml by setting this part did solve it:

    route:
      type: hidden
      default: **/news**

Still, after clicking on save I get the message: "Failed to save entry: %s"

But it is saved as expected... Maybe I can get a tip on this thread on this "Failed to save entry: %s" message?

**Comment by mahagr** [comment]
Did you update the site to the latest versions? As the issue is likely already fixed.

**Comment by w00fz** [comment]
Was this ever resolved?
